### PR TITLE
Add Flexera Software Vulnerability Research

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

**Comment by Carlgo11** [comment]
Can't it just be incorporated into the same entry?

**Comment by 40504** [comment]
this pr is based on suggestions from #5662

**Comment by Carlgo11** [comment]
Ah okay. I assumed that you would just reopen the PR when you had time to make the requested changes.

Anyways, Arun's reasoning for splitting the entires looks good to me so my previous comment can be discarded. I'll review this PR further tomorrow.

File: entries/a/app.flexera.com.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "Software Vulnerability Research": {
```
Comment (Carlgo11) [inline entries/a/app.flexera.com.json]: Should it be `Flexera Software Vulnerability Research` or `Flexera One`?

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/a/app.flexera.com.json (hunk 1 above)
Comment (40504) [reply entries/a/app.flexera.com.json]: @ApCoder123 suggested Software Vulnerability Research

Comment (ApCoder123) [reply entries/a/app.flexera.com.json]: Whoops. I did mean to write `Flexera Software Vulnerability Research` in my original review. Sorry for not making that clear.

Comment (Carlgo11) [reply entries/a/app.flexera.com.json]:
```suggestion
  "Flexera Software Vulnerability Research": {
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**